JumpTheQueue/index.asciidoc

```diff
@@ -0,0 +1,40 @@
+= Tutorial for the Jump The Queue Application
+====
+Jump The Queue is a small application based on the devonfw framework, which you can create yourself by following our simple step-by-step tutorial. By doing so, you will learn about the app development workflow and gain insight into the design of a professional business information system.
+
+More information about Jump The Queue on https://github.com/devonfw/jump-the-queue
+====
+
+Install devonfw IDE
+[step]
+--
+installDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+Clone the Jump-The-Queue Repository.
+[step]
+--
+cloneRepository("", "https://github.com/devonfw/jump-the-queue.git")
+--
+
+====
+[step]
+== Build and Run Java Server
+--
+buildJava("jump-the-queue/java/jtqj", true)
+runServerJava("jump-the-queue/java/jtqj/server", { "startupTime": 40, "port": 8081, "path": "jumpthequeue" })
+--
+Now the Java Server should be running.
+====
+
```

Comment:
You have to change the path path to the backend server in the environment.ts file. Otherwise the use cannot login and test the jump the queue application.

Reply:
I'm not quite sure what you mean?

Reply:
At the moment the user can start the app by clicking on the link in the last step, but cannot use it because there is no connection to the java backend. So you have to adjust the 'baseUrlRestServices' property in the 'jump-the-queue/angular/src/environments/environment.ts' file and insert the correct URL to the java backend server.